.github/workflows/stale-pr.yml

```diff
@@ -0,0 +1,55 @@
+name: Stale PR Management
+
+on:
+  schedule:
+    - cron: '0 0 * * *' 
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  stale:
+    name: Mark stale PRs
+    runs-on: ubuntu-latest
+    
+    permissions:
+      pull-requests: write
+    
+    steps:
+      - name: Run stale action
+        uses: actions/stale@v9
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          
+          days-before-stale: 14
+          days-before-close: 30
+          
+          stale-pr-message: |
+            This pull request has been inactive for 2 weeks and will be marked as stale.
+            
+            If you're still working on this, please:
+            - Update the PR with recent changes
+            - Respond to any review comments
+            - Rebase on the latest main branch
+            
+            The PR will be closed in 30 days if there's no activity.
+            
+            If you need help or have questions, feel free to ask! We appreciate your contribution. 🙏
+          
+          close-pr-message: |
+            This pull request was automatically closed due to inactivity.
+            
+            If you'd like to continue this work, feel free to:
```

Comment:
Same thing here, only one option is necessary. Though they should always feel free to comment if they need assistance :)